Src/ILGPU/Runtime/Cuda/CudaAccelerator.cs

```diff
@@ -449,6 +449,13 @@ public CudaCacheConfiguration CacheConfiguration
         protected override AcceleratorStream CreateStreamInternal() =>
             new CudaStream(this);
 
+        /// <summary>
+        /// Creates a CUDA stream with blocking calls
+        /// </summary>
+        /// <returns>The created CUDA stream</returns>
+        public CudaStream CreateBlockingStream() =>
```

Comment:
QUESTION: Should we expose a more generic API?
e.g. `public CudaStream CreateStream(StreamFlags flags)`

QUESTION: Should we expose `CreateStream(IntPtr)` to allow the caller to provide an externally created stream?

Reply:
@MoFtZ I think we should also expose the `CreateStream(IntPtr)` constructor. This extends the possibilities to use low-level Cuda API interop.